gtest_parallel_tests.py

```diff
@@ -446,5 +446,207 @@ def root():
                                    os.path.join(root(), 'c', 'd', 'bin'),
                                    'Test.case', 1))
 
+
+class TestFindTests(unittest.TestCase):
+  class MockOptions(object):
+    def __init__(self, failed=False, gtest_also_run_disabled_tests=False,
+                 gtest_color="False", gtest_filter="", output_dir="", repeat=1,
+                 shard_count=1, shard_index=1):
+      self.failed = failed
+      self.gtest_also_run_disabled_tests = gtest_also_run_disabled_tests
+      self.gtest_color = gtest_color
+      self.gtest_filter = gtest_filter
+      self.output_dir = output_dir
+      self.repeat = repeat
+      self.shard_count = shard_count
+      self.shard_index = shard_index
+
+  class MockTestTimes(object):
+    def __init__(self, test_data=None):
```

Comment:
Can this be `test_data={}` or will that refer to the same dictionary across invocations?